worldedit-bukkit/adapters/adapter-1_19/src/main/java/com/sk89q/worldedit/bukkit/adapter/impl/fawe/v1_19_R1/regen/PaperweightRegen.java

```diff
@@ -364,7 +364,7 @@ protected void cleanup() {
             Fawe.instance().getQueueHandler().sync(() -> {
                 try {
                     freshChunkProvider.close(false);
-                } catch (IOException e) {
+                } catch (Exception e) {
```

Comment:
Why was this changed to a more generic exception?

Reply:
Pre-Chunk Update the method throws an IOException, after the update, no exception is thrown anymore :/